admin_manual/maintenance/moving-data-folders.rst

```diff
@@ -0,0 +1,75 @@
+============================
+How To Move a Data Directory
+============================
+
+If you need to move your ownCloud data directory from its current location to
+somewhere else, here is a manual process that you can take to make it happen.
+
+.. NOTE:: 
+   This example assumes that:
+
+   - The current folder is: `var/www/owncloud/data`
+   - The new folder is: `/mnt/owncloud`
+   - You’re using Apache as your webserver
```

Comment:
ah, so that's how you do apostrophes in RST. Last time I used the regular one it didn't like it.